gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,303 @@
+
+###################################
+NVIDIA GPU DRA Driver
+###################################
+
+*******************************
+About the NVIDIA GPU DRA Driver
+*******************************
+
+The NVIDIA GPU DRA Driver leverages the Kubernetes Dynamic Resource Allocation (DRA) API to support NVIDIA IMEX channels available in GH200 and GB200 GPUs.
+The NVIDIA GPU DRA Driver creates and manages IMEX channels through the creation of a ComputeDomain custom resource. Use this custom resource to define your resource templates, and then reference the templates within your workload specs.
+
+An IMEX channel is a construct that allows a set of GPUs to directly read and write each other's memory over a high-bandwidth NVLink. 
+The NVLink connection may either be directly between GPUs on the same node or between GPUs on separate nodes connected by an NVSwitch. 
+Once an IMEX channel has been established for a set of GPUs, they are free to read and write each other's memory via extensions to the CUDA memory call APIs.
+
+The ability to support IMEX channels on GH200 and GB200 systems is essential, as they have been designed specifically to exploit the use of IMEX channels to turn a rack of GPU machines (each with a small number of GPUs) into a giant supercomputer with up to 72 GPUs communicating at full NVLink bandwidth. 

```

Comment:
I think it is imp to first summarize what is to follow. That imex channels are meant for multi-node communication

IMEX channel (by definition) is a resource that spans multiple nodes, hence the ability to support IMEX channels on GH200 and GB200 systems is essential. These GH200 and GB200 systems are designed  specifically to leverage the use of IMEX channels to turn a rack of GPU machines (each with a small number of GPUs) into a giant supercomputer with up to 72 GPUs communicating at full NVLink bandwidth.